src/App/Fossa/VPS/Scan/Core.hs

```diff
@@ -66,67 +51,13 @@ createRevisionLocator projectName organizationId revision = do
 sherlockInfoEndpoint :: Url 'Https -> Url 'Https
 sherlockInfoEndpoint baseurl = baseurl /: "api" /: "vendored-package-scan" /: "sherlock-info"
 
--- /api/vendored-package-scan/ci
-createProjectEndpoint :: Url 'Https -> Url 'Https
-createProjectEndpoint baseurl = baseurl /: "api" /: "vendored-package-scan" /: "ci"
-
--- /api/vendored-package-scan/ci/complete
-completeProjectEndpoint :: Url 'Https -> Url 'Https
-completeProjectEndpoint baseurl = baseurl /: "api" /: "vendored-package-scan" /: "ci" /: "complete"
-
--- /api/vendored-package-scan/project-scan-filters/:locator
-projectScanFiltersEndpoint :: Url 'Https -> Locator -> Url 'Https
-projectScanFiltersEndpoint baseurl locator = baseurl /: "api" /: "vendored-package-scan" /: "project-scan-filters" /: unLocator locator
-
 {-
   FIXME: Every function below this line is using a data structure designed for the CLI.
   This tightly couples us to our CLI API, and is very tedious to change with the merge of `vpscli` and `fossa` exe's.
 -}
```

Comment:
Since we removed the functions that this comment was referring to (except one), we can remove the comment as well.